api/feature-flags.js

```
@@ -0,0 +1,59 @@
+
+// Function to save feature flags to file
+const saveFeatureFlags = () => {
+  try {
+    fs.writeFileSync(FEATURE_FLAGS_PATH, JSON.stringify(featureFlags, null, 2));
+  } catch (error) {
+    console.error('Failed to save feature flags:', error);
+  }
+};
+
+router.get('/feature-flags', authenticateRequest, (req, res) => {
+  if (featureFlags.hasOwnProperty(feature)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)